additional_recipes/ros-humble-dynmsg/recipe.yaml

```diff
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+package:
+  name: ros-humble-dynmsg
+  version: "1.0.0"
```

Comment:
Hopefully the package will be soon also released upstream (see https://github.com/osrf/dynamic_message_introspection/issues/29). I know that the package.xml there reports 1.0.0, but to avoid confusion with a possible future 1.0.0, could we use some smaller number like 0.0.1 or 0.1.0 ? Thanks!